src/rust/bitbox02-rust/src/workflow/mnemonic_c_unit_tests.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright 2024 Shift Crypto AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub use super::cancel::Error as CancelError;
+use super::confirm;
+
+use alloc::string::String;
+use alloc::string::ToString;
+
+pub async fn show_and_confirm_mnemonic(words: &[&str]) -> Result<(), CancelError> {
+    let _ = confirm::confirm(&confirm::Params {
```

Comment:
not sure what the purpose is of the contents of this function - it could just be empty right?